mem0/embeddings/fastembed.py

```diff
@@ -0,0 +1,26 @@
+from embedding.base import EmbeddingBase
```

Comment:
This would lead to import errors. 
I added the minor fixes in #1548  and  #1574 

Suggested change:
```python
from mem0.embeddings.base import EmbeddingBase
```

Reply:
Thanks. [`4665f5b` (#1579)](https://github.com/mem0ai/mem0/pull/1579/commits/4665f5b73a98d180a99f80e7bf5a45126729b294).